### DISCUSSION TITLE
Mode presets for specific applications

### DISCUSSION THREAD

**Discussion by BlueCricket971** [seerge/g-helper#1698] [Ideas]
Would it be possible to have a specific mode activate when running a specific game or application? For example if I load up a game, turbo mode would automatically turn on. Or if I have a streaming service open, silent would activate. I think this would be a pretty useful feature as switching modes would automatically be done instead of having to manually do it when you open a specific app/game.

**Comment by seerge** [comment]
@BlueCricket971 hello, I was answering this [question already](https://github.com/seerge/g-helper/issues/1080), it's quite difficult to do + game detection is often unreliable
